wordpress-mobile/WordPress-iOS
# Reader: Check for finite value before setting offset

This PR addresses a crash in the reader detail that is not well understood.

We've received a number of reports of the app crashing when being opened. Crash reports
show that setContentOffset is being passed a non-finite value, leading to the crash. This
PR checks for a finite value before making the assignment, hopefully avoiding the crash.

We should be mindful of subsquent reports of layout issues with the reader detail after
opening the app. These could provide a clue as to the underlying cause.

Fixes #10571

To test:
Launch the reader. 
Check the reader detail layout. 
Change the apps orientation to trigger `viewWillTransitionToSize:with:`.  
Confirm the detail layout rotates correctly and there is no crash.
Bonus: Set a breakpoint and confirm that `setContentOffset` is being called during rotation.
